src/connect/ConnectWorkspaceFolder.ts

```diff
@@ -398,12 +398,7 @@ export class ConnectWorkspaceFolder extends WorkspaceFolderBase {
 
         try {
             // Send POST request to http://localhost:<control-port>/api/remoting/stop.
-            await request.post({
-                method: `POST`,
-                uri: `http://localhost:${this._controlPort.toString()}/api/remoting/stop/`,
-                body: ``,
-                simple: true
-            });
+            await got.post(`http://localhost:${this._controlPort.toString()}/api/remoting/stop/`);
```

Comment:
Is this change required to make tests pass? Does the `got` library allow mock http requests?

Reply:
this is part of another PR , I worked on top of it. Sorry for the confusion. Here is the link https://github.com/Azure/vscode-bridge-to-kubernetes/pull/61